package.json

```diff
@@ -7,15 +7,15 @@
   "scripts": {
     "bootstrap": "yarn",
     "bootstrap:ci": "yarn install --frozen-lockfile",
-    "build:all": "yarn build:crypto-dependencies && lerna run build",
-    "build:ci": "lerna run build --since origin/main --include-dependencies",
-    "build:clients:generic": "lerna run --scope '@aws-sdk/aws-echo-service' --include-dependencies build",
-    "build:clients:since:release": "yarn build:packages && lerna run build $(lerna changed | grep -e '@aws-sdk/[client|lib]-*' | sed 's/^/ --scope /' | tr '\n' ' ')",
-    "build:crypto-dependencies": "lerna run --scope '@aws-sdk/{types,util-utf8,util-locate-window,hash-node}' --include-dependencies build",
```

Comment:
Removed deprecated packages @aws-sdk/util-utf8 and @aws-sdk/hash-node from build:crypto-dependencies scope to fix errors resolving package paths.